services/movie_session.py

```diff
@@ -1,3 +1,4 @@
+from django.core.exceptions import ObjectDoesNotExist
```

Comment:
The import statement for `ObjectDoesNotExist` is correct, but ensure that you handle exceptions properly in your functions to avoid potential runtime errors.